packages/ui/src/components/vuestic-components/va-backtop/VaBacktop.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div
+    v-if="visible"
+    class="va-backtop"
+    :style="computedStyle"
+    @click="scrollToTop()"
+  >
+    <slot>
```

Comment:
Don't any references provide default styling? I would imagine primary CTO button with arrow to be good enough for significant number of users.